magento/cart2quote/2017-02-01.yaml

```diff
@@ -0,0 +1,13 @@
+
+title:     Remote Code Execution in Qquoteadv/controllers/DownloadController.php
+link:      https://cart2quote.zendesk.com/hc/en-us/articles/115000616303--FIXED-Security-Vulnerability-in-downloadCustomOptionAction
+cve:       ~
+branches:
+    4:
+        time:     2017-02-01 10:45:00
+        versions: ['>=4.1.6', '<5.0'
+    5:
+        time:     2017-02-01 10:45:00
+        versions: ['>=5.0.1', '<5.4.4']
+reference: composer://cart2quote/cart2quote
```

Comment:
It looks like composer is used for the Magento 2 installation, which is not affected by this issue. I'm not sure how to best represent this. 

https://cart2quote.zendesk.com/hc/en-us/articles/115000616303

> All Magento 1 version

https://www.cart2quote.com/documentation/magento-2-cart2quote-installation-manual/

(See the title)